fe2-android/app/src/main/res/values/strings.xml

```diff
@@ -310,6 +310,8 @@
   <string name="error_post_transaction">Could not post a transaction\n%1$s</string>
   <string name="error_no_rollcall_closed_in_LAO">Error no RollCall closed in the LAO\n%1$s</string>
   <string name="error_public_key_receiver">Error on the receiver s public key\n%1$s</string>
+  <string name="qr_code_not_pop_token">Error, this QR code is not your pop token</string>
+  <string name="qr_code_not_main_pk">Error, this QR code is not your main public key</string>
```

Comment:
Same here maybe? Just not sure who the "your" is referring to since the person reading this message will be the one scanning, not the one presenting the wrong qr code 😅

Reply:
True I hadn't thought about that

Reply:
maybe "is not a pop token" ?

Reply:
👍